### PR TITLE
Input

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
project live - https://solutionground.netlify.app/ (not updated)
image

### CODE REVIEW

File: input/Input/Input.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    margin-top: 2em;
+    width: auto;
+}
+
+.text-input{
+    position: relative;
+    width: 90%;
+    padding: 6px 0;
+    line-height: 16%;
+    font-size: 14px;
```
Comment (Mohasin-Haque) [inline input/Input/Input.css]: You can use rem if you want.

File: input/Input/Input.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    /* transition: all 0.2s linear; */
+}
+
+.text-input:focus + .text-placeholder {
+    transform: translateY(-4.1em);
+    opacity: 0.5;
+}
+
+.icon-password{
+  padding-left: 25px;
```
Comment (Mohasin-Haque) [inline input/Input/Input.css]: Same can be done everywhere where you applied px.. If you have time then you can change it to relative units.

File: input/Input/Input.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    position:absolute;
+    right:50px;
+    top:6px;
+}
+
+.text-input-date{
+    position: relative;
+    width: 90%;
+    padding: 6px 0;
+    /* line-height: 16%; */
```
Comment (Mohasin-Haque) [inline input/Input/Input.css]: remove commented part

File: input/Input/Input.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+
+  // var number = document.getElementById("number");
+  // var password = document.getElementById("password");
+  // var email = document.getElementById("email");
+  // var tel = document.getElementById("tel");
+  // var url = document.getElementById("url");
+  // var search = document.getElementById("search");
+  
+
```
Comment (Mohasin-Haque) [inline input/Input/Input.js]: you can remove the commented part

File: input/Input/Input.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    }
+  }
+
+ 
+
+
+  
+
+
+
```
Comment (Mohasin-Haque) [inline input/Input/Input.js]: Remove the whitespaces

File: input/index.css [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+
+:root {
+  --icon-primary-color:#594adb ;
+  --icon-secondary-color: #f17c0e;
+    /* --icon-primary-color:#5468ff; darkblue*/
+      /* --icon-primary-color:#990000 ; 
+  --icon-secondary-color: #FFCC00; */ 
+
+  /* --icon-secondary-color : #7264e8; #f17c0e*/
```
Comment (Mohasin-Haque) [inline input/index.css]: Remove the commented part from all the files if not needed

File: input/index.css [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+  --yellow:#ffc107;
+  --blue:#1793b8;
+  --red:#dc3545;
+  --green:#28a745;
+  --lightblack :#212529;
+  --skyblue: #87ceeb;
+  
+ 
+
+
```
Comment (Mohasin-Haque) [inline input/index.css]: Please run prettier or format the code

**Review by Mohasin-Haque — COMMENTED** [review]
Your code is good but it needs formatting and removal of commented parts for sanity. Please go through the comments and also run prettier to it.

File: input/Input/Input.css (hunk 1 above)
Comment (chanchal-panpaliya) [reply input/Input/Input.css]: done

File: input/Input/Input.css (hunk 2 above)
Comment (chanchal-panpaliya) [reply input/Input/Input.css]: done

File: input/Input/Input.css (hunk 3 above)
Comment (chanchal-panpaliya) [reply input/Input/Input.css]: done

File: input/Input/Input.js (hunk 4 above)
Comment (chanchal-panpaliya) [reply input/Input/Input.js]: done

File: input/Input/Input.js (hunk 5 above)
Comment (chanchal-panpaliya) [reply input/Input/Input.js]: done

File: input/index.css (hunk 6 above)
Comment (chanchal-panpaliya) [reply input/index.css]: done

File: input/index.css (hunk 7 above)
Comment (chanchal-panpaliya) [reply input/index.css]: done

**Review by HarshnaNHaswani — COMMENTED** [review]
The interactions are pretty good, nice implementation. My only suggestion  is that you work n color contrast and line heights

**Pull request merged**